dotnet/samples/KernelSyntaxExamples/Example65_HandlebarsPlanner.cs

```diff
@@ -3,42 +3,51 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
-using System.Linq;
-using System.Security.Cryptography;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.SemanticKernel;
 using Microsoft.SemanticKernel.Diagnostics;
+using Microsoft.SemanticKernel.Functions.OpenAPI.Extensions;
 using Microsoft.SemanticKernel.Planners.Handlebars;
+using Plugins.DictionaryPlugin;
 using RepoUtils;
 
 /**
  * This example shows how to use the Handlebars sequential planner.
  */
 public static class Example65_HandlebarsPlanner
 {
-    private static int s_sampleCount;
+    private static int s_sampleIndex;
+
+    private const string CourseraPluginName = "CourseraPlugin";
 
     /// <summary>
     /// Show how to create a plan with Handlebars and execute it.
     /// </summary>
     public static async Task RunAsync()
     {
-        s_sampleCount = 0;
-        Console.WriteLine($"======== {nameof(Example65_HandlebarsPlanner)} ========");
+        s_sampleIndex = 1;
+        bool shouldPrintPrompt = true;
 
+        // Using primitive types as inputs and outputs
         await PlanNotPossibleSampleAsync();
-        await RunDictionarySampleAsync();
+        await RunDictionaryWithBasicTypesSampleAsync();
         await RunPoetrySampleAsync();
         await RunBookSampleAsync();
+
+        // Using Complex Types as inputs and outputs
+        await RunLocalDictionaryWithComplexTypesSampleAsync(shouldPrintPrompt);
+
+        // TODO (@teresaqhoang): Enable this sample once complex types are supported for remote plugins return type.
+        // await RunRemoteDictionaryWithComplexTypesSampleAsync();
```

Comment:
Can we leave this out until it's working?

Reply:
Done